multiversx_sdk_cli/cli_shared.py

```diff
@@ -382,6 +374,25 @@ def prepare_guardian_account(args: Any) -> IAccount:
     return account
 
 
+def load_sender_account(args: Any) -> Union[IAccount, None]:
+    hrp = config.get_address_hrp()
+
+    if args.pem:
+        return Account.new_from_pem(file_path=Path(args.pem), index=args.pem_index, hrp=hrp)
```

Comment:
Just a question: when loading a key from a PEM file, shouldn't we learn the HRP directly from the comment within the PEM file (i.e. the _label_)? Oh, I don't think the SDK is wired that way anyway :thought_balloon:

Reply:
Indeed, the sdk is not wired that way.